src/qml/WelcomeScreen.qml

```diff
@@ -266,6 +267,108 @@ Page {
     }
   }
 
+  // Sparkles & unicorns
+  Rectangle {
+    anchors.fill: parent
+    color: "#00000000"
+    visible: imageDialLogo.value < 0.1
+
+    MouseArea {
+      id: mouseArea
+      anchors.fill: parent
+      acceptedButtons: Qt.LeftButton | Qt.RightButton
+      hoverEnabled: true
+      propagateComposedEvents: true
+      onReleased: mouse.accepted = false
+      onDoubleClicked: mouse.accepted = false
+      onPressAndHold: mouse.accepted = false
+      onClicked:  {
+        burstSomeSparkles(mouse.x, mouse.y)
+        mouse.accepted = false
+      }
+      onPressed:  {
+        burstSomeSparkles(mouse.x, mouse.y)
+        mouse.accepted = false
+      }
+      onPositionChanged: {
+        burstSomeSparkles(mouse.x, mouse.y)
+        mouse.accepted = false
+      }
+    }
+
+    ParticleSystem {
+      id: particles
+      running: imageDialLogo.value < 0.1
+    }
+
+    ParticleSystem {
+      id: unicorns
+      running: imageDialLogo.value < 0.1
+    }
+
+    ImageParticle {
+      anchors.fill: parent
+      system: particles
+      source: "qrc:///particleresources/star.png"
```

Comment:
@domi4484 should this file be present in this pr?

Reply:
No, this is provided by Qt sinc V5.2.
https://doc.qt.io/qt-5/qml-qtquick-particles-imageparticle.html